BeamMonitor: Options `particles` and `beam_moments` (#1557)
BLAST-ImpactX/impactx: docs/source/usage/python.rst

   ``json`` only works with serial/single-rank jobs.
   By default, the first available backend in the order given above is taken.

   openPMD `iteration encoding <https://openpmd-api.readthedocs.io/en/latest/usage/concepts.html#iteration-and-series>`__ determines if multiple files are created for individual output steps or not.

>    Variable based encoding requires linear read access (``Access.read_linear``) in downstream readers.

Author: ```suggestion
   Variable based is an `experimental feature with ADIOS2 <https://openpmd-api.readthedocs.io/en/0.14.0/backends/adios2.html#experimental-new-adios2-schema>`__ and currently requires linear read access (``Access.read_linear``) in downstream readers.
```
